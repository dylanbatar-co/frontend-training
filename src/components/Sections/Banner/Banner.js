import React from 'react';
import Button from 'emerald-ui/lib/Button';
export const Banner = () => {
  return (
    <section
      className="banner mt-5 mb-5"
      style={{
        backgroundImage: `url(${process.env.PUBLIC_URL}/assets/img/banner.jpg)`
      }}
    >
      <div className="banner-content container">
        <h2 className="mb-2">Subscribe to our newsletter</h2>
        <p className="mb-4">
          Subscribe to our newsletter to receive weekly digest of the best and
          most ground-breaking news. Also receive a discount on your monthly
          subscription.
        </p>

        <Button color="default" shape="outline" className="banner-button">
          Subscribe
        </Button>
      </div>
    </section>
  );
};
