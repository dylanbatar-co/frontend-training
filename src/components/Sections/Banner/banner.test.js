import { shallow } from 'enzyme';
import { Banner } from './Banner';

describe('Banner section', () => {
  it('Section snapshot', () => {
    const component = shallow(<Banner />);
    expect(component).toMatchSnapshot();
  });
});
