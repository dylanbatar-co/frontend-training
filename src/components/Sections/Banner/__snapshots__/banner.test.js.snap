// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner section Section snapshot 1`] = `
<section
  className="banner mt-5 mb-5"
  style={
    Object {
      "backgroundImage": "url(/assets/img/banner.jpg)",
    }
  }
>
  <div
    className="banner-content container"
  >
    <h2
      className="mb-2"
    >
      Subscribe to our newsletter
    </h2>
    <p
      className="mb-4"
    >
      Subscribe to our newsletter to receive weekly digest of the best and most ground-breaking news. Also receive a discount on your monthly subscription.
    </p>
    <Button
      className="banner-button"
      color="default"
      displayName="Button"
      shape="outline"
      size="xs"
      type="button"
    >
      Subscribe
    </Button>
  </div>
</section>
`;
